velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,24 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns error status if the date is invalid. We treat days of the previous
+/// or next months as a part of the specified WEEK_OF_MONTH. For example, if
```

Comment:
WEEK_OF_MONTH -> weekOfMonth for consistency

Reply:
Thanks. Done.